apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -65,20 +66,24 @@ export function ColumnToggleButton() {
         store.selectedColumns,
         store.setSelectedColumns,
     ]);
-    const [open, setOpen] = useState(false);
+    const [anchorEl, setAnchorEl] = useState<HTMLElement>();
+    const open = useMemo(() => Boolean(anchorEl), [anchorEl]);
```

Comment:
```suggestion
    const open = Boolean(anchorEl)
```
This isn't wrong, but I don't think it's worth memo-izing. Good job for being aware of this, you're becoming a true React developer! 💯 

Another fun :tm: part of React is having incurring mental overhead when deciding whether or not to memo-ize things because you don't know how many times it'll be recalculated and realizing that you'll still get a poorly optimized app anyways.